### PR TITLE
Issue: renamed the INFORMATION level to MESSAGE

### PR DESCRIPTION
**Pull request by agarny**
Addresses issue #674.

### CODE REVIEW

File: src/api/libcellml/logger.h [hunk 1]
```diff
@@ -143,26 +143,27 @@ class LIBCELLML_EXPORT Logger
...
     /**
-     * @brief Get issue of level INFORMATION at the specified @p index.
+     * @brief Get issue of level MESSAGE at the specified @p index.
      *
-     * Returns a piece of information at the @p index.  If the @p index
-     * is not valid a @c nullptr is returned, the valid range for the @p index
-     * is [0, \#informations).
+     * Returns a message at the @p index.  If the @p index is not valid a
+     * @c nullptr is returned, the valid range for the @p index is
+     * [0, \#messagess).
```
Comment (kerimoyle) [inline src/api/libcellml/logger.h]: messssagesss, my preciousssss ;)

**Review by kerimoyle — COMMENTED** [review]
Sooo close ...

**Comment by agarny** [comment]
> Sooo close ...

Argh! :)

File: src/api/libcellml/logger.h (hunk 1 above)
Comment (agarny) [reply src/api/libcellml/logger.h]: Fixed in commit 08d76b08.

**Review by kerimoyle — APPROVED** [review]
Wahoo :)

**Comment by agarny** [comment]
> Wahoo :)

Yeah, success! :)

**Review by nickerso — APPROVED** [review]

**Pull request merged**